tools/render-test/render-d3d11.cpp

```diff
@@ -612,165 +578,149 @@ void* D3D11Renderer::map(ID3D11Buffer * buffer, MapFlavor flavor)
     // We update our constant buffer per-frame, just for the purposes
     // of the example, but we don't actually load different data
     // per-frame (we always use an identity projection).
-    D3D11_MAPPED_SUBRESOURCE dxMapped;
-    HRESULT hr = dxContext->Map(dxBuffer, 0, dxMapFlavor, 0, &dxMapped);
+    D3D11_MAPPED_SUBRESOURCE mappedSub;
+    HRESULT hr = m_immediateContext->Map(buffer, 0, mapType, 0, &mappedSub);
     if (FAILED(hr))
         return nullptr;
 
-    return dxMapped.pData;
+    return mappedSub.pData;
 }
 
 void* D3D11Renderer::map(Buffer* buffer, MapFlavor flavor)
 {
     return map(((BufferImpl*)buffer)->buffer, flavor);
 }
 
-void D3D11Renderer::unmap(ID3D11Buffer * buffer)
+void D3D11Renderer::unmap(ID3D11Buffer* buffer)
 {
-    auto dxContext = dxImmediateContext;
-    dxContext->Unmap(buffer, 0);
+    m_immediateContext->Unmap(buffer, 0);
 }
 
-void D3D11Renderer::unmap(Buffer* buffer)
+void D3D11Renderer::unmap(Buffer* bufferIn)
```

Comment:
My preference for parameter naming conventions (I'm realizing at this point maybe I need to go ahead and write up a coding convention document rather than leave this all implicit and have us all argue over it...):

- No "Hungarian notation" stuff. Ever.
- If it is unclear in context whether a pointer/reference parameter is being used for input, output, or both, then its name can be prefixed with `in`, `out`, or `io`
- The above should not in general apply to parameters that are pointers to "reference types" as opposed to "value types," where the distinction between the two is hopefully clear from the type itself